hvantk/ancestry/pipeline.py

```
@@ -0,0 +1,1034 @@
+        types=types,
+        missing="",
+    )
+    ht = ht.key_by(key)
+
+    # Register cleanup for when Python exits (temp files cleaned up by OS anyway)
+    def cleanup():
+        import shutil
+        try:
+            shutil.rmtree(tmp_dir, ignore_errors=True)
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)